### PR TITLE
test(appeals): e2e test for uploading an updated version of a doc

### PR DESCRIPTION
**Pull request by RHARV018**
e2e test for uploading an updated version of a doc on the appellant case

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by james-hill-kc — APPROVED** [review]
Really interesting to have this insight into how the E2E tests work, thanks @RHARV018 - all looks good to me! It would be nice to include an assertion to confirm the file was uploaded, but I don't think there's realistically a way to do that. It might be worth checking that the relevant row on the case details page contains a list item for the uploaded file.

**Comment by RHARV018** [comment]
Thanks @james-hill-kc , I did think about adding an assertion but at the moment it doesn't display the most recent doc version in the row on appellant case page - it always just shows the original file name.

File: appeals/e2e/cypress/e2e/back-office-appeals/manageDocsAppellantCase.spec.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+describe('manage docs on appellant case', () => {
+	beforeEach(() => {
+		cy.login(users.appeals.caseAdmin);
+	});
+
+	it('upload new version of document on appellant case', { tags: tag.smoke }, () => {
+		cy.createCase().then((caseRef) => {
+			happyPathHelper.uploadDocAppellantCase(caseRef);
+			cy.reload();
```
Comment (PeterBridgerKC) [inline appeals/e2e/cypress/e2e/back-office-appeals/manageDocsAppellantCase.spec.js:24]: Why is this required?
